packages/popover/src/vaadin-popover.js

```diff
@@ -119,16 +151,20 @@ class Popover extends PopoverPositionMixin(
         theme="${ifDefined(this._theme)}"
         .positionTarget="${this.target}"
         .position="${effectivePosition}"
-        .opened="${this._opened}"
+        .opened="${this.opened}"
         .modeless="${!this.modal}"
         .focusTrap="${this.modal}"
         .withBackdrop="${this.withBackdrop}"
         ?no-horizontal-overlap="${this.__computeNoHorizontalOverlap(effectivePosition)}"
         ?no-vertical-overlap="${this.__computeNoVerticalOverlap(effectivePosition)}"
         .horizontalAlign="${this.__computeHorizontalAlign(effectivePosition)}"
         .verticalAlign="${this.__computeVerticalAlign(effectivePosition)}"
+        @mouseenter="${this.__onOverlayMouseEnter}"
+        @mouseleave="${this.__onOverlayMouseLeave}"
+        @focusin="${this.__onOverlayFocusin}"
+        @focusout="${this.__onOverlayFocusout}"
         @opened-changed="${this.__onOpenedChanged}"
-        restore-focus-on-close
+        .restoreFocusOnClose="${this.__isTrigger('click') && this.trigger.length === 1}"
```

Comment:
We discussed this internally and agreed to remove this line for now since the focus restoration will be addressed separately.

Reply:
Removed this logic for now.